staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/types.go

```diff
@@ -48,6 +47,9 @@ type ListOptions struct {
 	ResourceVersion string
 	// Timeout for the list/watch call.
 	TimeoutSeconds *int64
+
+	Limit    int64
```

Comment:
For query params 0 and unspecified have the same behavior other than the possibility we'd reject (which I don't think we should do, since that would mean a new client running against a server with paging disabled would then fail).